client/TextArea.jsx

```diff
@@ -73,14 +74,9 @@ const TextArea = () => {
         />
         <div onChange={set("submission")}>
           <label htmlFor="submission">
-            <input
-              type="radio"
-              name="submission"
-              value="offer"
-              defaultChecked={true}
-            />{" "}
-            Offer
-            <input type="radio" name="submission" value="request" /> Request
+          <input aria-label="Offer" type="radio" name="submission" value="offer" defaultChecked={true}/> Offer
+          <input aria-label="Request" type='radio' name='submission' value='request'/> Request
```

Comment:
issue: Each of these inputs need their own `label` element.
issue: Each of these inputs need their own `id`.
issue: Since each radio button should have its own label, neither of these actually need an `aria-label`. It's generally best to use `aria-label`s when you have no way of showing the necessary text on the screen.
issue: Neither of these radio buttons need the text string at the end of the line, since they should each have their own label.

I've made a [demonstration CodePen](https://codepen.io/novellac/pen/RwebLRy) here you can check out.